src/components/CardItem.js

```diff
@@ -0,0 +1,23 @@
+import './CardItem.css';
+
+const CardItem = ({ url, title, description, createdAt, imageSource }) => {
+  return (
+    <div className="CardItem">
+      <div className="starArea">
+        <div className="starButton" type="button"></div>
```

Comment:
버튼 태그가 되어야 할 것 같아요~ 
안에 별모양 아이콘도 추후에 추가되면 될 것 같아요!
```suggestion
        <button className="starButton" type="button"></button>
```